internal/testutils/systembus.go

```diff
@@ -0,0 +1,126 @@
+package testutils
+
+import (
+	"context"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sync"
+	"testing"
+
+	"github.com/godbus/dbus/v5"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	sdbus sync.Once
+
+	sdbusMU                sync.Mutex
+	nbRunningTestsSdbus    uint
+	stopDbus               context.CancelFunc
+	dbusCmd                *exec.Cmd
+	config                 string
+	savedDbusSystemAddress string
+)
+
+// StartLocalSystemBus allows to start and set environment variable to a local bus, preventing polluting system ones.
+func StartLocalSystemBus() func() {
```

Comment:
Maybe add it as a helper function? This way, we would have access to `require` assertions and other stuff.

Reply:
Ack, maybe we can work together to export the `testutils` package as it is really becoming a staple in our projects.